internal/backoff/backoff.go

```diff
@@ -26,16 +27,13 @@ import (
 )
 
 // RetriableError explicitly instructs Backoff to retry.
-type RetriableError string
-
-// Error returns string representation of the retriable error.
-func (re RetriableError) Error() string {
-	return string(re)
+type RetriableError struct {
+	error
 }
```

Comment:
Would `type RetriableError error` work, or you would need to explicitly add the `Error()` method?

Reply:
I did try that first.  It breaks `errors.As` unfortunately.